brayns/common/engine/Engine.cpp

```diff
@@ -74,23 +83,48 @@ Renderer& Engine::getRenderer()
 
 bool Engine::continueRendering() const
 {
+    auto frameBuffer = _frameBuffers[0];
     return _parametersManager.getAnimationParameters().getDelta() != 0 ||
-           (_frameBuffer->getAccumulation() &&
-            (_frameBuffer->numAccumFrames() <
+           (frameBuffer->getAccumulation() &&
+            (frameBuffer->numAccumFrames() <
              _parametersManager.getRenderingParameters().getMaxAccumFrames()));
 }
 
-void Engine::_writeFrameToFile()
+void Engine::addFrameBuffer(FrameBufferPtr frameBuffer)
+{
+    _frameBuffers.push_back(frameBuffer);
+}
+
+void Engine::removeFrameBuffer(FrameBufferPtr frameBuffer)
+{
+    auto i = std::find_if(_frameBuffers.begin(), _frameBuffers.end(),
+                          [&frameBuffer](auto x) { return frameBuffer == x; });
+    if (i != _frameBuffers.end())
+        _frameBuffers.erase(i);
```

Comment:
doesn't just _frameBuffers.erase(frameBuffer); do the right thing?

Reply:
It's a `std::vector`, so unfortunately, no.